modules/central-services-private-config.adoc

```diff
@@ -14,7 +14,6 @@ The credentials required for pulling images from your registry.
 
 [NOTE]
 ====
-* If you are using the `stackrox.io` registry, you must specify the `imagePullSecrets.username` and `imagePullSecrets.password` parameters.
 * If you are using a custom registry, you must also specify the `image.registry` parameter.
```

Comment:
I think this should say something like
```suggestion
* If you are using a custom registry, you must specify the `imagePullSecrets.username`, `imagePullSecrets.password` and `image.registry` parameters.
```

Moreover, I suspect that only on OpenShift you don't need `imagePullSecrets.username` and `imagePullSecrets.password` for default `registry.redhat.io` because OpenShift authenticates globally to it.
On non-OpenShift Kubernetes `imagePullSecrets.username` and `imagePullSecrets.password` may be needed for `registry.redhat.io`. However, this needs to be tested as I'm not fully confident.

Reply:
Well, I suggest to a) apply you removal to `If you are using the `stackrox.io` registry` line, b) apply my changes to `If you are using a custom registry` line. We will not be wrong this way.

The point about necessity of `imagePullSecrets.username` and `imagePullSecrets.password` on non-OpenShift Kubernetes can remain unaddressed for now.

I think, it is worth to also mention shut down of image registries hosted at `stackrox.io` and `collector.stackrox.io` addresses somewhere in release notes. https://srox.slack.com/archives/C3W3U0RGW/p1657047197148959
It's too late to include in the release but at least the online documentation will be clear about the shutdown.